torch/_inductor/config.py

```diff
@@ -430,6 +430,9 @@ class aot_inductor:
     # If not specified, a temp directory will be created under the default caching path
     output_path = ""
 
+    # Wether to codegen abi compatible model.so
+    abi_compatible = True  # is_fbcode()
```

Comment:
hard coded to True, I am working on adding test copy to test both and will change this back